configs/platforms/debian-9-amd64.rb

```diff
@@ -6,7 +6,7 @@
 
   plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/debian/pl-build-tools-release-#{plat.get_codename}.deb"
   plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get update -qq; apt-get install -qy --no-install-recommends build-essential devscripts make quilt pkg-config debhelper rsync fakeroot"
-  plat.install_build_dependencies_with "DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends "
+  plat.install_build_dependencies_with "DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends --force-yes"
```

Comment:
Is it ok to `--force-yes` in these platform files? This is done so that development repos of PE puppet-agent can be installed during pe-bolt-server's provisioning step. I might be able to find somewhere else to put this if necessary.

Reply:
Does this need an answer?

Reply:
If someone knows a reason I shouldn't put this here, yeah, but I suspect it doesn't matter